.codex/hooks/log_stop.py

```
@@ -0,0 +1,60 @@
+import subprocess
+import sys
+
+
+def utc_now():
+    return datetime.datetime.now(datetime.timezone.utc).isoformat().replace("+00:00", "Z")
+
+
+def repo_root(cwd):
+    try:
+        result = subprocess.run(
+            ["git", "rev-parse", "--show-toplevel"],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)